lib/screens/UsersProfileScreen.dart

```diff
@@ -0,0 +1,1576 @@
+                                  ? globalDecoration.copyWith(
+                                  enabledBorder: const OutlineInputBorder(
+                                      borderSide:
+                                      BorderSide(color: Colors.red)),
+                                  suffixIcon: const Icon(Icons.clear,
+                                      color: Colors.red),
+                                  hintText: 'Confirm Password')
+                                  : globalDecoration.copyWith(hintText: 'Confirm Password'),
+                              style: const TextStyle(
+                                fontSize: 20,
+                                color: Colors.black,
+                              ),
+                              textAlign: TextAlign.left,
+                              textInputAction: TextInputAction.done,
+                              onChanged: (password) {
+                                if (password.isEmpty) {
+                                  setState(() => unfilledConfirmPassword = true);
+                                }
+                                else {
+                                  if (_password.value.text.isEmpty) {
+                                    errorMessage = 'Passwords must match!';
+                                  }
+                                  else if (passwordsMatch()) {
+                                    errorMessage = '';
+                                    unfilledConfirmPassword = false;
+                                  }
+                                  setState(() {});
+                                }
+                              },
+                            ),
+                          ),
+                          Container(
+                            margin: const EdgeInsets.only(bottom: 20),
+                            child: Text(
+                              errorMessage,
+                              style: const TextStyle(fontSize: 14, color: Colors.red),
+                              textAlign: TextAlign.center,
+                            ),
+                          ),
+                          Container(
+                            width: 180,
+                            //padding: EdgeInsets.all(15),
+                            child: ElevatedButton(
+                              onPressed: () async {
+                                if (areFieldsValid()) {
+                                  if (passwordsMatch()) {
+                                    String firstName = _firstName.value.text.trim();
+                                    String lastName = _lastName.value.text.trim();
+                                    String username = _email.value.text.trim();
+                                    String changes = '{"firstname": "$firstName","lastname": "$lastName","lastSeen": ${user
+                                        .lastSeen}"username": "$username","password": "${user
```

Comment:
I think `lastSeen` is computed server side, I don't think you need to worry about including that